images/dnsmasq/Dockerfile

```diff
@@ -1,13 +1,12 @@
-FROM centos:7
+FROM centos:8
```

Comment:
We didn't ?

hmmm... but we still need to be ask about this package, so we can use a UBI image instead. /cc: @fatherlinux

Reply:
iirc most container images are using rhel7 ubi, it would be more consistent to keep using centos:7 even if there will be no sharing. But both `centos:7` and `centos:8` are fine with me.